src/Snippets/Docs/Chaos.Outcome.cs

```diff
@@ -103,17 +106,139 @@ public static void OutcomeGeneratorDelegates()
                     {
                         < 100 => Outcome.FromResult(new HttpResponseMessage(HttpStatusCode.InternalServerError)),
                         < 150 => Outcome.FromResult(new HttpResponseMessage(HttpStatusCode.TooManyRequests)),
-                        < 250 => Outcome.FromResult(CreateResultFromContext(args.Context)),
+                        < 250 => Outcome.FromResult(new HttpResponseMessage(CreateResultFromContext(args.Context))),
```

Comment:
As far as I can see this `OutcomeGenerator` delegate could be marked as `static`